kivy/ext/mpl/backend_kivyagg.py

```diff
@@ -0,0 +1,216 @@
+"""
```

Comment:
You should replace this documentation with your own documentation specific to kivyagg.